desktop/apps/auction2/client/reducers.js

```diff
@@ -30,6 +30,7 @@ const initialState = {
   followedArtistRailPage: 1,
   followedArtistRailSize: 4,
   initialMediumMap: [],
+  isOpen: sd.AUCTION && sd.AUCTION.is_open,
```

Comment:
Ah, yeah I guess this was just trying to be defensive and also makes testing easier since. It _should_ be there.